tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml

```diff
@@ -148,16 +148,16 @@
             </when>
             <when value="mitgcm-resultplots">
                 <param name="netcdf_input_file" label="NetCDF input file" optional="false" help="The NetCDF input file that should be plotted. It must comply to some specific properties with respect to variables. (URL must be stored in a .txt file)" type="data" format="txt"/>
-                <param name="variable" label="Variable to be plotted" optional="false" help="The variable that should be plotted. You can choose between 'T', 'S', 'U', 'V', 'Eta' and 'W'." type="select">
+                <param name="variable" label="Variable to be plotted" optional="false" value="T" help="The variable that should be plotted. You can choose between 'T', 'S', 'U', 'V', 'Eta' and 'W'." type="select">
                     <option value="T">T</option>
                     <option value="S">S</option>
                     <option value="U">U</option>
                     <option value="V">V</option>
                     <option value="Eta">Eta</option>
                     <option value="W">W</option>
                 </param>
-                <param name="plot_time" label="Hour to be plotted" optional="false" help="The time of day (hourly resolution) that should be plotted. Please specify it in this specific format: '20220703-12:00:00'. We assume that the hour is in the simulation period." type="text"/>
-                <param name="plot_depth" label="Depth to be plotted" optional="false" help="The depth at which should be plotted. The plot will be at the closest vertical level to this depth. Please specify a negative value. We assume the depth is in the range of our depths." type="float"/>
+                <param name="plot_time" label="Hour to be plotted" optional="false" value="YYYYMMDD-HH:MM:SS" help="The time of day (hourly resolution) that should be plotted. Please specify it in this specific format: '20220703-12:00:00'. We assume that the hour is in the simulation period." type="text" />
```

Comment:
This will cause user errors I'm pretty sure that people will run it with the YYYYMMDD letters.

Reply:
Alright, replaced it by a validator